src/main/java/Check.java

```diff
@@ -0,0 +1,72 @@
+public class Check {
+     //Function to check if an input has mark as the first word
+     public static boolean isMark(String input){
```

Comment:
Consider using a more descriptive name for code readability 
```suggestion
     public static boolean isMarkCommand(String input) {
```
I have noticed the same issue in other places too.